account_asset_analytic/tests/test_account_asset.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountAsset(TransactionCase):
+    def setUp(self):
+        super(TestAccountAsset, self).setUp()
+        self.category = self.env.ref(
```

Comment:
I prefer to create all needed test data in the test to avoid migration issues. See these comments: https://github.com/OCA/bank-payment/pull/270#issuecomment-222094139